content/en/docs/setup/cri.md

```diff
@@ -45,7 +45,7 @@ Use the following commands to install Docker on your system:
     stable"
 
 ## Install docker ce.
-apt-get update && apt-get install docker-ce=18.06.0~ce~3-0~ubuntu
+apt-get update && apt-get install docker-ce=18.06.2~ce~3-0~ubuntu
```

Comment:
https://github.com/kubernetes/website/pull/12591/files#diff-9b5d940e9fc0e3fab81c81aad0dc300fR23
please change this value to 18.06.2 too.

`Version 18.06.2 is recommended, ....`